scripts/minter.ts

```diff
@@ -158,11 +162,53 @@ async function setWhitelist(whitelist: string) {
   );
 }
 
+// Takes config.minter address and config.startTime
+// and tries to update existing minter start time.
+// Can not change if public mint already started.
+async function updateStartTime() {
+  const client = await getClient();
+  const account = toStars(config.account);
+  const minter = toStars(config.minter);
+
+  const answer = await inquirer.prompt([
+    {
+      message:
+        'Are you sure your want to change public mint start time to ' +
+        config.startTime +
+        ' ?',
+      name: 'confirmation',
+      type: 'confirm',
+    },
+  ]);
+  if (!answer.confirmation) return;
+
+  // time expressed in nanoseconds (1 millionth of a millisecond)
+  const publicStartTime: Timestamp = (
+    new Date(config.startTime).getTime() * 1_000_000
+  ).toString();
+
+  const result = await client.execute(
+    account,
+    minter,
+    { update_start_time: publicStartTime },
+    'auto',
+    'update public mint start time'
```

Comment:
Not sure we need the memo here... extra space / gas, and we already know it's update start time from the message.